benchmarks/templates/benchmarks/leaderboard/leaderboard-table.html

```diff
@@ -83,6 +83,12 @@
                     <i class="fa-solid fa-down-left-and-up-right-to-center"></i>
                   </span>
                 {% endif %}
+                {# info icon and box #}
+                {% if "Deng2009-top1" in benchmark.identifier %}
```

Comment:
To work in production, this line needs to be:

`{% if "ImageNet-top1" in benchmark.identifier %}`

If you were using fixtures when developing this, then the benchmark identifier might be different then the 2.0 identifier (those fixtures need to be updated). When I checked this branch out, everything worked as expected when the the `if` statement checks for the `identifier` in the suggested change above